sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/reflect/DoFnInvokers.java

```diff
@@ -43,136 +36,4 @@
   }
 
   private DoFnInvokers() {}
-
-  /**
-   * Returns a {@link DoFnInvoker} for the given {@link Object}, which should be either a {@link
-   * DoFn} or an {@link OldDoFn}. The expected use would be to deserialize a user's function as an
-   * {@link Object} and then pass it to this method, so there is no need to statically specify what
-   * sort of object it is.
-   *
-   * @deprecated this is to be used only as a migration path for decoupling upgrades
-   */
-  @Deprecated
-  public static DoFnInvoker<?, ?> invokerFor(Serializable deserializedFn) {
```

Comment:
Alternatively, you'll need to port the Dataflow service to no longer call this method, which is presumably now possible. But synchronizing those efforts is not necessary, in my view.